srcStatic/OP2Memory.cpp

```diff
@@ -21,24 +20,20 @@ void SetLoadOffset()
 
 bool Op2MemCopy(void* destBaseAddr, void* sourceAddr, int size)
 {
-	DWORD oldAttr;
-	DWORD ignoredAttr;
-	BOOL bSuccess;
 	void* destAddr = (void*)(loadOffset + (int)destBaseAddr);
 
-	// Try to unprotect the memory
-	bSuccess = VirtualProtect(destAddr, size, PAGE_EXECUTE_READWRITE, &oldAttr);
+	// Try to unprotect memory
+	DWORD oldAttr;
+	BOOL bSuccess = VirtualProtect(destAddr, size, PAGE_EXECUTE_READWRITE, &oldAttr);
 	if (!bSuccess) {
-		char buffer[64];
-		_snprintf_s(buffer, sizeof(buffer), "Op2MemCopy: Error unprotecting memory at: %x", reinterpret_cast<unsigned int>(destAddr));
-		PostErrorMessage(__FILE__, __LINE__, buffer);
-		return false;	// Abort if failed
+		PostErrorMessage(__FILE__, __LINE__, "Error unprotecting memory at: " + std::to_string(reinterpret_cast<unsigned int>(destAddr)));
```

Comment:
Just noticed this will produce a change in the output. The "%x" modifier prints the address in hex, which is far more valuable than the decimal equivalent that [`std::to_string`](https://en.cppreference.com/w/cpp/string/basic_string/to_string) would produce. I do not know of an equivalent that can produce hex output. The closest is using the [`std::hex`](https://en.cppreference.com/w/cpp/io/manip/hex) I/O manipulator with a [`std::stringstream`](https://en.cppreference.com/w/cpp/io/basic_stringstream).